index.js

```diff
@@ -0,0 +1,149 @@
+const compareLangKeys = () => {
+  // lib is needed to show deuplicated keys
+  const jsonValidator = require('json-dup-key-validator');
+
+  //requiring path and fs modules
+  const path = require('path');
+  const fs = require('fs');
+  const directoryPath = path.join(__dirname, 'locales');
+
+  const mapAllFiles = (files, filesContent) => {
+    const isJsonFile = (JSONdata, file) => {
```

Comment:
Functions named `isSomething`, `areSomething`, etc. are expected to return boolean values. Consider renaming this function to mean what it does.